packages/d3fc-chart/src/cartesianBase.js

```diff
@@ -45,38 +43,15 @@ export default (setPlotArea, defaultPlotArea) =>
                 container.select('.y-label-container>.y-label')
                     .text(yLabel);
 
-                container.select('.top-label')
-                    .style('margin-top', '2em');
-
-                container.enter()
-                    .append('div')
-                    .attr('class', 'chart-label')
-                    .style('grid-column', 3)
-                    .style('-ms-grid-column', 3)
-                    .style('grid-row', 1)
-                    .style('-ms-grid-row', 1)
-                    .style('height', '2em')
-                    .style('line-height', '2em')
-                    .style('text-align', 'center');
-
-                container.select('.chart-label')
-                    .text(chartLabel(data));
-
                 decorate(container, data, index);
             });
 
             selection.call(cartesian);
         };
 
         rebindAll(cartesianBase, cartesian, include(/^x/, /^y/));
+        rebind(cartesianBase, cartesian, 'chartLabel');
```

Comment:
Can you do it in one line?

```javascript
rebindAll(cartesianBase, cartesian, include(/^x/, /^y/, 'chartLabel'));
```

Reply:
I was wondering about that but I wasn't sure.